src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -47,7 +47,7 @@ namespace intel_cpu {
  */
 static constexpr Property<bool> denormals_optimization{"CPU_DENORMALS_OPTIMIZATION"};
 
-static constexpr Property<float> sparse_weights_decompression_rate{"SPARSE_WEIGHTS_DECOMPRESSION_RATE"};
+static constexpr Property<float> sparse_weights_decompression_rate{"CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE"};
```

Comment:
I suppose it's not a BW compatible change if users used string name?

Reply:
Unfortunately, yes, I didn't fix it after merging my previous PR https://github.com/openvinotoolkit/openvino/pull/13775

Reply:
so, why do we need to change the name now?

Reply:
because it's a bug that's hasn't been fixed. Currently, we can't use c++ or python api to set this property, we will get error " [ NOT_FOUND ] Unsupported property SPARSE_WEIGHTS_DECOMPRESSION_RATE by CPU plugin"